governance/polls/Ratification Poll for Adding Events Core Unit (EVENTS-001) - April 11, 2022.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Ratification Poll for Adding Events Core Unit (EVENTS-001) - April 11, 2022
+summary: These subproposals onboard Events Core Unit (EVENTS-001) whose mandate is to coordinate and execute MakerDAO branded events at Crypto/DEV Conference locations.
+discussion_link: https://forum.makerdao.com/t/mip39c2-sp32-adding-events-core-unit-events-001/13780
+vote_type: Plurality Voting
+categories:
+   - MIPs
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2022-04-11T16:00:00
+end_date: 2022-04-25T16:00:00
+---
+# Ratification Poll for Adding Events Core Unit (EVENTS-001) - April 11, 2022
+
+The Governance Facilitators have placed a ratification poll into the [voting system](https://vote.makerdao.com/polling) as part of the responsibilities defined in [MIP51](https://mips.makerdao.com/mips/details/MIP51). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for fourteen days beginning on Monday, April 11 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to Adding Events Core Unit (EVENTS-001) being accepted and implemented in the Maker Protocol.
+
+### MIP39c2-SP32: Adding events Core Unit (EVENTS-001)
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*The Events Core Unit will handle planning, organization, and execution of MakerDAO branded events. Our vision is to use an exhaustive list of themes, visuals, set dressing, technology and branding to carry out events to the highest standards for the MakerDAO community.*
+
+Please review the links below to inform your position on this proposal before voting.
+* [Canonical Proposal Version](https://github.com/makerdao/mips/commit/1a75df31c022c3049a2f623ed3178a10457ff5fb)
```

Comment:
Use this link: https://github.com/makerdao/mips/blob/1a75df31c022c3049a2f623ed3178a10457ff5fb/MIP39/MIP39c2-Subproposals/MIP39c2-SP32.md